Makefile

```diff
@@ -1,15 +1,17 @@
-ALL = $(shell ls services | grep -v -e elasticsearch- -e php- -e percona) elasticsearch php
+ALL = $(shell ls services | grep -v -e elasticsearch- -e php- -e percona) elasticsearch php php-fpm-nginx
```

Comment:
If we renamed the directory to `nginx-php-fpm` I think we could avoid any modifications to the makefile. Thoughts?

Reply:
I'm guessing you preferred php at the beginning because the tags will follow php? Not sure how important that is.

Reply:
Yeah, that's it exactly. But I'm open either way.

Reply:
The one slightly odd thing about the tags is that they also have `fpm` in them, so to get 8.2.x of php, you need to use `tugboatqa/php-fpm-nginx:8.2-fpm`, instead of `tugboatqa/php-fpm-nginx:8.2`. I suppose we could modify the getTags function to also do the filtering, and then strip the fpm from the tag.

Reply:
Thinking about this some more, what if we did something like how the apache tags work and add a `-nginx` suffix: `php:8.2-fpm-nginx`

Reply:
I like that, because it also follows the variant pattern other images use, such as `node:18-alpine`.

Reply:
> Thinking about this some more, what if we did something like how the apache tags work and add a -nginx suffix: php:8.2-fpm-nginx

I investigated this, and it would take a refactor of our scripts to accomplish. So I went instead with the approach in #112